sos/report/plugins/opensvc.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class Opensvc(Plugin, IndependentPlugin):
+
+    short_desc = 'OpenSVC cluster and services (config and state collection)'
+    plugin_name = 'opensvc'
+    profiles = ('cluster', 'services', 'system')
+    packages = ('opensvc',)
+
+    def get_status(self, kind):
+        getobjs = self.collect_cmd_output("om %s ls --color=no" % kind)
+        dirname = kind + '_status'
+        if getobjs['status'] == 0:
+            for line in getobjs['output'].splitlines():
+                self.add_cmd_output([
+                    "om %s print status --color=no" % line
+                ], subdir=dirname)
```

Comment:
`add_cmd_output()` takes a single item as well as a list, so we can just pass the bare string here for each line iteration

Reply:
Fixed in commit https://github.com/sosreport/sos/pull/2441/commits/4c7d1b1665ef2ba81690411a6ea1014abfce92b9